src/store/middlewares/settings.js

```diff
@@ -19,7 +19,16 @@ const settings = store => next => (action) => {
       if (action.data.token && action.data.token.active !== token.active) {
         store.dispatch(emptyTransactionsData());
       }
-      setInStorage('settings', store.getState().settings);
+      setInStorage('settings', {
+        ...store.getState().settings,
+        token: {
+          active: 'LSK',
+          list: {
+            BTC: false,
+            LSK: true,
+          },
+        },
+      });
```

Comment:
At this place token settings should already equal to this. If there is some case were it is not in my opinion would be better to set the correct store during startup. At `settingsRetrieved` in middleware if `BTC=true` I would override the localstorage and get sure reducer does not receive `BTC=true`.

Reply:
Fixed. However, I implemented it in actions since the settings are retrieved there and it might be better to override it there rather than dispatching the same action in the middleware